sympy/integrals/tests/test_integrals.py

```diff
@@ -2160,3 +2161,10 @@ def test_integral_issue_26566():
 
     # Assert that the symbolic result matches the correct value
     assert simplify(numeric_symbolic_result - numeric_correct_result) == 0
+
+def test_issue_27231():
+    # https://github.com/sympy/sympy/issues/27231
+    i2 = Integral(sqrt(1 - 0.5625*(x + 0.333333333333333)**2),(x,-1,1))
+    result=i2.doit().evalf()
+    expected_result = nsimplify(i2).doit().evalf()
+    assert abs(result - expected_result) < 1e-8
```

Comment:
I merged the other PR because the test here does not state what the actual expected result of the integral should be.

Reply:
It's okay, no problem. I will improve myself for next time